atspi-common/src/events/mod.rs

```diff
@@ -131,7 +103,7 @@ impl Default for EventBodyQT {
 /// Standard event body (GTK, `egui`, etc.)
 /// NOTE: Qt has its own signature: [`EventBodyQT`].
 /// Signature `(siiva{sv})`,
-#[derive(Clone, Debug, Serialize, Deserialize, Type, PartialEq)]
+#[derive(Debug, Serialize, Deserialize, Type, PartialEq)]
```

Comment:
Why was the `Clone` derive removed here?

Reply:
`EventBodyOwned` has fields of type `OwnedValue`. 
In zvariant 4, `Value`s no longer have `Clone` because of the [safe Fd work](https://github.com/dbus2/zbus/pull/395). Cloning is now achieved through [`Value::try_clone`](https://docs.rs/zvariant/latest/zvariant/enum.Value.html#method.try_clone).

I manually re-implemented `Clone` using `OwnedValue::try_clone` to not break our API, because: 
- Suddenly missing `Clone` is pretty rough.
- There is only a remote chance that we will hit the error path in this context, because it requires the current process to exceed the maximum number of open files: (from `Value::try_clone` docs)

```text
Errors

This method can currently only fail on Unix platforms for [Value::Fd](https://docs.rs/zvariant/latest/zvariant/enum.Value.html#variant.Fd) variant containing an [Fd::Owned](https://docs.rs/zvariant/latest/zvariant/enum.Fd.html#variant.Owned) variant. This happens when the current process exceeds the maximum number of open file descriptors.
```